src/main/java/com/softserveinc/ita/pageobjects/ProductPage.java

```diff
@@ -87,5 +87,24 @@ public ProductPage switchPhotoTo(int index) {
     public String getImgSource() {
         return $x("//img[@class = 'picture-container__picture']").getAttribute("src");
     }
+
+    @Step("ProductPage: Switched kit to '{index}'")
+    public ProductPage swithKitTo(int index) {
+        $x(format("(//section[@class='kits product-kits ng-star-inserted']//button[contains(@class,'slider-dots__button')])[%s]", index)).click();
+
+        return this;
+    }
+
+    public int getMainKitProductPrice(int index) {
+        return parseIntPrice($x(format("//li[@class = 'simple-slider__item ng-star-inserted'][%s]//p[@class = 'kits-tile__price']", index)).shouldBe(visible).getText());
```

Comment:
extract into variable and pass to parseIntPrice()

var mainKitProductPrice = $x(format("//li[@class = 'simple-slider__item ng-star-inserted'][%s]//p[@class = 'kits-tile__price']", index))
.shouldBe(visible)
.getText();